source/Grid/Grid.js

```diff
@@ -427,6 +428,12 @@ export default class Grid extends Component {
           let columnDatum = this._columnMetadata[columnIndex]
           let renderedCell = renderCell({ columnIndex, rowIndex })
           let key = `${rowIndex}-${columnIndex}`
+
+          // any other falsey value will be rendered
+          // as a text node by React
+          if (renderedCell == null || renderedCell === false)
```

Comment:
Curious what prompted this change. Do people actually render empty cells? :) I guess it doesn't hurt to support this use-case. It's just not one I've considered.